ang/civicase/CaseCard.html

```diff
@@ -0,0 +1,109 @@
+<!-- To do: Use variables and add functionality in the updated HTML.
+<div ng-if="!bulkAllowed" class="lockDiv">
+  <div class="fa fa-lock"></div>
+</div>
+<div class="case-list-checkbox" ng-if="!viewingCase && selectedCases && bulkAllowed">
+  <input type="checkbox" ng-model="item.selected" />
+  <label></label>
+</div>
+<div class="case-list-info">
+  <h4>{{ item.client[0].display_name }}</h4>
+  <div class="btn-group btn-group-sm" ng-if="item.client.length > 1">
+    <button type="button" class="btn btn-info dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" title="{{ ts('Additional clients') }}">
+      +{{ item.client.length - 1 }}
+    </button>
+    <ul class="dropdown-menu" >
+      <li ng-repeat="(index, client) in item.client" ng-if="index">
+        <span>{{ client.display_name }}</span>
+      </li>
+    </ul>
+  </div>
+  <div ng-if="item.client[0].email">
+    <strong>{{ ts('Email:') }}</strong>
+    {{ item.client[0].email }}
+  </div>
+  <div ng-if="item.client[0].phone">
+    <strong>{{ ts('Phone:') }}</strong>
+    {{ item.client[0].phone }}
+  </div>
+  <div>
+    <strong>{{ ts('Case ID:') }}</strong>
+    {{ item.id }} &nbsp;
+    <span class="civicase-tags">
+      <span
+        class="label label-default"
+        title="{{ tag['tag_id.description'] }}"
+        ng-repeat="tag in item.tag_id || []"
+        ng-class="{'text-dark': CRM.utils.colorContrast(tag['tag_id.color'] || '#cccccc') === 'black'}"
+        style="background-color: {{ tag['tag_id.color'] }};"
+      >{{ tag['tag_id.name'] }}</span>
+    </span>
+  </div>
+  <div class="case-activity-summary-icons" ng-if="bulkAllowed">
+    <a class=""
+       ng-repeat="category in activityCategories"
+       ng-href="#{{ activityFeedUrl(item.id, category.name, 'incomplete') }}"
+       ng-class="{'item-overdue': item.activity_summary[category.name][0].is_overdue}"
+       ng-if="item.category_count.incomplete[category.name]"
+       title="{{ ts('%1 upcoming %2', {1: item.category_count.incomplete[category.name], 2: category.label}) }}"
+    >
+      <i class="fa {{ category.icon }}"></i>
+      <span class="badge noti-badge">{{ item.category_count.incomplete[category.name] }}</span>
+    </a>
+  </div>
+</div>
+ -->
+
+ <div class="panel-default panel-strip-primary civicase__case-card civicase__animated-checkbox-card">
```

Comment:
`panel` class is still missing.

Reply:
Sorry, I forgot to mention. Panel class is adding extra border-radius of 2px and margin-bottom of 20px which we don't need in the styles. That's why we are not using panel class here.

Reply:
ok, but we need to follow the styleguide of `panel-strip-primary`. And that includes the panel class. Once that is done, we can override any styles which we dont need.